ecom_core/config/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+class DateModel(models.Model):
+    created = models.DateTimeField()
+    updated = models.DateTimeField()
+    
+    class Meta:
+        abstract = True
+
```

Comment:
You should add only one blank line at the end of the file. But here you have added 2 blank lines at the end of the file.